README.md

```diff
@@ -93,6 +105,11 @@ jake help             # List the above commands.
 node built/local/tsc.js hello.ts
 ```
 
+or
+
+```bash
+make usage

```

Comment:
`node built/local/tsc.js hello.ts` is an example of how to compile a TypeScript file in this case `hello.ts`.  There is not `hello.ts` in the repo so run `make usage` will fail.  Also, it's not very useful to compile `hello.ts` as people will probably want to compile files with different names as well.

Reply:
Removed